.circleci/config.yml

```diff
@@ -0,0 +1,45 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/node:8.12.0
+    working_directory: ~/lansingcodes/lubot
+    environment:
+      TZ: America/Detroit
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - v1-dep-{{ .Branch }}-
+            - v1-dep-master-
+            - v1-dep-
+      - run:
+          command: |
+            npm install
+            npm test
+      - save_cache:
+          key: v1-dep-{{ .Branch }}-{{ epoch }}
```

Comment:
Do you mind changing this to:

```
key: base-deps-v1-{{ .Branch }}-{{ checksum "package-lock.json" }}
```

This will allow us to more directly correlate caching to dependency changes.

Reply:
that makes a lot of sense. I'll make that change